doc/classes/OmniLight3D.xml

```diff
@@ -14,8 +14,11 @@
 	</tutorials>
 	<members>
 		<member name="omni_attenuation" type="float" setter="set_param" getter="get_param" default="1.0">
-			The light's attenuation (drop-off) curve. A number of presets are available in the [b]Inspector[/b] by right-clicking the curve. Zero and negative values are allowed but can produce unusual effects.
-			[b]Note:[/b] Very high [member omni_attenuation] values (typically above 10) can impact performance negatively if the light is made to use a larger [member omni_range] to compensate. This is because culling opportunities will become less common and shading costs will be increased (as the light will cover more pixels on screen while resulting in the same amount of brightness). To improve performance, use the lowest [member omni_attenuation] value possible for the visuals you're trying to achieve.
+			Controls the distance attenuation function for omnilights.
+			[b]Note:[/b] A value of [code]0[/code] smoothly attenuates light at the edge of the range. A value of [code]1[/code] approaches a physical lighting model. A value of [code]0.5[/code] approximates linear attenuation.
+			[b]Note:[/b] Setting it to [code]1[/code] may result in distant objects receiving minimal light, even within range. For example, with a range of [code]4096[/code], an object at [code]100[/code] units receives less than [code]0.1[/code] energy.
+			[b]Note:[/b] If your lighting range exceeds 200, it's recommended to use 0; for ranges below 200, consider using 0.5.
+			[b]Note:[/b] Using negative or higher values may lead to unexpected results.
```

Comment:
The line below is continuing the train of thought of the above line. They could be one in the same, perhaps. Let's surround
```suggestion
			[b]Note:[/b] If your lighting range exceeds [code]200[/code], it's recommended to use [code]0[/code]. Otherwise, consider using [code]0.5[/code]. Using negative or higher values may lead to unexpected results.
```

Reply:
I'm a bit unsure about "lighting range", I'd say "light range" reads better